CMakeLists.txt

```diff
@@ -25,7 +25,9 @@ set(VERSION "1.6.0")
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-set(CMAKE_INSTALL_PREFIX /opt/rocm/mivisionx)
+
+set(ROCM_PATH /opt/rocm CACHE PATH "default rocm installation path")
+set(CMAKE_INSTALL_PREFIX /opt/rocm/mivisionx CACHE PATH "default installation path")
```

Comment:
set(CMAKE_INSTALL_PREFIX /opt/rocm/mivisionx CACHE PATH "default installation path") needs to change to

```
set(CMAKE_INSTALL_PREFIX ${ROCM_PATH}/mivisionx CACHE PATH "default MIVisionX installation path")
```

Reply:
It is updated, please check the update changes.